src/Symfony/Component/Routing/Tests/Generator/Dumper/CompiledUrlGeneratorDumperTest.php

```diff
@@ -267,66 +272,64 @@ public function testAliases()
 
     public function testTargetAliasNotExisting()
     {
-        $this->expectException(RouteNotFoundException::class);
-
         $this->routeCollection->addAlias('a', 'not-existing');
 
-        file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());
+        $this->expectException(RouteNotFoundException::class);
 
-        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
+        file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());
 
-        $compiledUrlGenerator->generate('a');
+//        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
+//        $compiledUrlGenerator->generate('a');
     }
 
     public function testTargetAliasWithNamePrefixNotExisting()
     {
-        $this->expectException(RouteNotFoundException::class);
-
         $subCollection = new RouteCollection();
         $subCollection->addAlias('a', 'not-existing');
         $subCollection->addNamePrefix('sub_');
 
         $this->routeCollection->addCollection($subCollection);
 
-        file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());
+        $this->expectException(RouteNotFoundException::class);
 
-        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
+        file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());
 
-        $compiledUrlGenerator->generate('sub_a');
+//        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
+//        $compiledUrlGenerator->generate('sub_a');
```

Comment:
weird, it looks like the expected exception is thron in `$this->generatorDumper->dump()` 🤔 So the following code with the `CompiledUrlGenerator` is not needed, so maybe the test is not needed anymore or the test must be adjusted somehow to test the behavior of `CompiledUrlgenerator`....